cob_obstacle_distance/src/distance_manager.cpp

```diff
@@ -438,6 +425,12 @@ void DistanceManager::registerObstacle(const moveit_msgs::CollisionObject::Const
         return;
     }
 
+    if(msg->operation == msg->ADD && this->obstacle_mgr_->count(msg->id) > 0)
+    {
+        ROS_ERROR_STREAM("Element " << msg->id << " exists already. ADD not allowed!");
```

Comment:
According to http://docs.ros.org/jade/api/moveit_msgs/html/msg/CollisionObject.html

 > # Adds the object to the planning scene. If the object previously existed, it is replaced.

Thus this should be handled as "move"?